src/QuickMarcEditor/QuickMarcEditor.js

```diff
@@ -129,9 +129,13 @@ const QuickMarcEditor = ({
   const type = leader?.content?.[6] || '';
   const subtype = leader?.content?.[7] || '';
 
-  const saveFormDisabled = action === QUICK_MARC_ACTIONS.EDIT
-    ? pristine || submitting
-    : submitting;
+  const saveFormDisabled = useMemo(() => {
```

Comment:
I think `useMemo` is a little bit overhead here, because there are no huge re-calculations and it may cost more then improves. What about replace it with

`const saveFormDisabled = submitting ? true : pristine;`